config/logicsoft/ZDB5100.xml

```diff
@@ -0,0 +1,498 @@
+        Byte 3: Colour saturation, blue
+            0-255 Specifies the saturation for the blue LEDs (Default is 47).
+        Byte 4: LED control
+            0-255 LED indicator control - see the application note about this parameter (Default is 0).
+        Default: 50065152
+			</Help>
+    </Value>
+  </CommandClass>
+  <!-- COMMAND_CLASS_SWITCH_ALL SwitchAllCmd_Get is not supported -->
+  <CommandClass id="39">
+    <Compatibility>
+      <GetSupported>false</GetSupported>
+    </Compatibility>
+  </CommandClass>
+  <!-- MultiInstance Command Class. Note: If you do not need Dedicated Relay Control instance, change below mapping to "endpoints", then rejoin the node to your network. -->
+  <CommandClass id="96">
+    <Compatibility>
+      <MapRootToEndpoint>true</MapRootToEndpoint>
+    </Compatibility>
+  </CommandClass>
+  <!-- Multi Channel Association Groups -->
+  <CommandClass id="142">
+    <Compatibility>
+      <ForceInstances>true</ForceInstances>
+    </Compatibility>
+  </CommandClass>
+  <!-- The Association Group Information -->
+  <CommandClass id="133">
+    <Associations num_groups="14">
+      <Group index="1" label="Lifeline" max_associations="5"/>
+      <Group auto="true" index="2" label="Send Basic Report (On/Off) when button #1 is used" max_associations="5"/>
+      <Group index="3" label="Sends Binary Switch Set (On/Off) when button #1 is used" max_associations="5"/>
+      <Group index="4" label="Sends Multilevel Switch Set / Multilevel Switch Start Level Change / Multilevel Switch Stop Level Change when button #1 is used" max_associations="5"/>
+      <Group auto="true" index="5" label="Send Basic Report (On/Off) when button #2 is used" max_associations="5"/>
+      <Group index="6" label="Sends Binary Switch Set (On/Off) when button #2 is used" max_associations="5"/>
+      <Group index="7" label="Sends Multilevel Switch Set / Multilevel Switch Start Level Change / Multilevel Switch Stop Level Change when button #2 is used" max_associations="5"/>
+      <Group auto="true" index="8" label="Send Basic Report (On/Off) when button #3 is used" max_associations="5"/>
+      <Group index="9" label="Sends Binary Switch Set (On/Off) when button #3 is used" max_associations="5"/>
+      <Group index="10" label="Sends Multilevel Switch Set / Multilevel Switch Start Level Change / Multilevel Switch Stop Level Change when button #3 is used" max_associations="5"/>
+      <Group auto="true" index="11" label="Send Basic Report (On/Off) when button #4 is used" max_associations="5"/>
+      <Group index="12" label="Sends Binary Switch Set (On/Off) when button #4 is used" max_associations="5"/>
+      <Group index="13" label="Sends Multilevel Switch Set / Multilevel Switch Start Level Change / Multilevel Switch Stop Level Change when button #4 is used" max_associations="5"/>
+      <Group auto="true" index="14" label="Send Basic Report with information about the actual level for the dimmer" max_associations="5"/>
```

Comment:
Basic is not desirable, as it would trigger OZW to query all values (most likely).  Would be better do set auto on index 13 instead. Same for all the other groups above. 

But question is - Shouldn't Association Group 1 cover all these cases?

Reply:
That would make sense - again I was looking at the ZHC5010 which is a similar device from same manufacturer and in that config file the basic report was set to auto. It makes no difference in the ZWave log (PR updated with ZWave log). I would appreciate it if you could point me in the right direction. Thanks again.

Reply:
Group 1 covers all cases.